packages/signer-cli/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
the whole project is apahce 2.0 so can probably omit this

Reply:
ok